addons/mail/static/src/less/chatter.less

```diff
@@ -1,33 +1,88 @@
+@o-mail-chatter-gap: 10px;
+
+// Default chatter layout.
+// Extended in enterprise by form_view.less to achieve the side-chatter
 .o_chatter {
     width: 100%;
+    padding-top: 0!important;
+
+    .o_spinner {
+        text-align: center;
+        font-size: large;
+    }
+
+    .o_chatter_topbar {
+        height: @odoo-statusbar-height;
+        position: relative;
+        margin: @grid-gutter-width*0.5 -@grid-gutter-width*0.5 0;
+        padding: 0 @grid-gutter-width*0.5;
+        .o-flex-display();
```

Comment:
I started a document for LESS coding conventions here: https://github.com/odoo/odoo/wiki/LESS-coding-guidelines
I know, this is purely to annoy you with my crazy convention tastes again :smile: